src/hooks/useSelectorWeb.ts

```diff
@@ -8,6 +7,6 @@ export const useSelectorWeb = <ReturnValue>(
   selector: Selector<CommonState, ReturnValue>
 ) => {
   return useSelector((state: { clientStore: CommonState }) =>
-    isEmpty(state.clientStore) ? undefined : selector(state.clientStore)
+    selector(state.clientStore)
```

Comment:
Not sure this is a better option -- but maybe worth calling out, we *could* return `{}` instead of delaying render. We'd have to check downstream if the derived fields (a & b in the case of { a, b } = useSelectorWeb()) aren't undefined, but it might reduce UI headache if there is any on initial paint

Reply:
yes we actually tried this! but that actually breaks other components that arent expectind an object, like
```
const someString = useSelectorWeb(getThatString)
```

Reply:
of course. carry on